data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,454 @@ mission "Unfettered returning home"
+			`	Another one steps forward. "We will allow you to keep the outfits, provided you give the ship to us on Darkcloak."`
+				to display
+					not "flagship planet: Darkcloak"
+			`	Another one steps forward, "Since you so kindly came to Darkcloak to deliver the ship to us, we will allow you to buy back any outfits you may still desire, as a courtesy."`
+				to display
+					has "flagship planet: Darkcloak"
+			choice
+				`	(Let them have it.)`
+				`	(Take off, now!)`
+					depart
+			`	You and your crew exit what was briefly your ship, and you inform the local Unfettered militia that it is now theirs. The Unfettered that were assembling around your ship happily disperse, except for one - probably the leader, judging by the amount of weaponry on their person. They come to talk to you, weapons in hand but lowered. "We are happy you could see reason and decided to hand over the ship. In exchange, we have decided to provide you with a suitable replacement, that should hopefully allow you to exit our space safely.`
+			`	"That relic was not meant for you, but remember that there are other ways for you to prove yourself." The leader of these Unfettered continues with blatantly forced casualness. "Say, for example, bringing us a Jump Drive... or two... that for sure will earn you favor, and prove you can fight."`
+				to display
+					not "Unfettered Jump Drive 1: offered"
+			branch "has detainers"
+				"outfit: Korath Detainer" > 1
+			action
+				fine 5000000
+			`	As he takes a look at the ship manifest, he seems to notice something missing. "How dare you? You will incur a fine of <fine> for having kept that Korath technology to yourself."`
+				goto nodetain
+			label "has detainers"
+			action
+				outfit "Korath Detainer" -2
+			label nodetain
+			`	"For now, you have enough to worry about with all of the missiles that will be heading your way. If I were you I'd install a lot of jammers. You'll need them, ha!" A general laughter erupts, and they let you get comfortable in the new ship.`
+			action
+				give ship "Lightning Bug (Unfettered Shipyards)" "Good Will"
+				take ship "Anomalocaris"
+					id "hai legacy"
+
+
+
+mission "Unfettered Anomalocaris Lost"
+	landing
+	invisible
+	source
+		attributes "unfettered"
+	to offer
+		has "Hai Legacy 3: done"
+		not "ship model (all): Anomalocaris"
+	on offer
+		conversation
+			`You see the sun slowly disappear into the horizon as you land on this planet, indicating it is already night here. Shortly after exiting your ship, you see a group of Unfettered Hai sitting on the side of the path, eating corn around a campfire. One of them, probably the youngest, judging by their size, tries to get your attention by calling you repeatedly. "Captain, Captain! Can you tell me stories about how you used great cunning to beat a swarm of Lightning Bugs?" Another one, probably their parent, then speaks up. "Come, sit with us. I am sure there is much to be said about that great relic from our past."`
```

Comment:
I'd assume you're pretty much the only human in UHai space

The story of you beating them with an old ship would travel quickly, I think, and I imagine they use such stories as morals for kids - dont be overconfident in this case

Reply:
If that's the case, it would make more sense if this was less of a disney movie credit scene, and more of an odd-chance bump into a family. Right now it sounds like the type of epilogue for after the Unfettered campaign, not after a short 3 part mission where you're just a person who delivered something and beat someone in a duel. Also consider that this is only if you've lost the ship - something as grandiose as a heartful scene like this doesn't really suit a mission that simply is designed to check if you still have it or not.

Reply:
well I'm not that attached to it so definitely open to tweaking it, but I was assuming it'd take a while to lose the ship
mb make it be a timer of 360 days after that? and then this mission?

also its not checking if you have it or not, it only triggers after losing it